client-side/test/homePage.test.js

```diff
@@ -0,0 +1,72 @@
+
+import '@testing-library/jest-dom'; // הוסף את השורה הזו
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import HomePage from '../src/components/homePageComponent';
+
+describe('HomePage Component', () => {
+  beforeEach(() => {
+    render(
+      <BrowserRouter>
+        <HomePage />
+      </BrowserRouter>
+    );
+  });
+
+  test('renders HomePage component without crashing', () => {
+    expect(screen.getByText('Enhance Your Browsing Experience with Our Chrome Add-on')).toBeInTheDocument();
+  });
+
+  test('renders all section titles', () => {
+    expect(screen.getByText('Enhance Your Browsing Experience with Our Chrome Add-on')).toBeInTheDocument();
+    expect(screen.getByText('Personalize Your Profiles')).toBeInTheDocument();
+    expect(screen.getByText('Manage Your Settings')).toBeInTheDocument();
+    expect(screen.getByText('Get Detailed Reports')).toBeInTheDocument();
+  });
+
+  test('renders all section descriptions', () => {
+    expect(screen.getByText('Install our Chrome extension to block or limit access to specific websites. Create profiles for work, home, and more, giving you control over your online activity.')).toBeInTheDocument();
+    expect(screen.getByText('Create and manage profiles for different settings like work and home. Link accounts from over 17,000 financial institutions and view your transactions in one place.')).toBeInTheDocument();
+    expect(screen.getByText('Customize your preferences to better track and understand your spending habits. Adjust your profile settings to suit your needs and enhance your browsing experience.')).toBeInTheDocument();
+    expect(screen.getByText('Review your transactions, track your spending by category, and receive monthly insights to help you better understand your money habits.')).toBeInTheDocument();
+  });
+
+  test('renders all buttons with correct text and links', () => {
+    expect(screen.getByText('Login')).toBeInTheDocument();
+    expect(screen.getByText('Go to Profiles')).toBeInTheDocument();
+    expect(screen.getByText('Go to Settings')).toBeInTheDocument();
+    expect(screen.getByText('Go to Reports')).toBeInTheDocument();
+
+    // בדוק שהלינקים מפנים לכתובת הנכונה
```

Comment:
please remove all the comments in Hebrew, if you want to add comments for explanation, it is great but they should be in English.